Jenkinsfile

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env groovy
+
+node('rhel7'){
+	stage('Checkout repo') {
+		deleteDir()
+		git url: 'https://github.com/redhat-developer/vscode-openshift-tools.git'
+	}
+
+	stage('Install requirements') {
+		def nodeHome = tool 'nodejs-8.11.1'
+		env.PATH="${env.PATH}:${nodeHome}/bin"
+		sh "npm install -g typescript vsce"
+	}
+
+	stage('Build') {
+		sh "npm install"
+		sh "npm run vscode:prepublish"
+	}
+
+	withEnv(['JUNIT_REPORT_PATH=report.xml']) {
+        stage('Test') {
+    		wrap([$class: 'Xvnc']) {
+    			sh "npm test --silent"
+    			junit 'report.xml'
+    		}
+        }
+	}
+
+	stage('Package') {
+        def packageJson = readJSON file: 'package.json'
+        sh "vsce package -o openshift-${packageJson.version}-${env.BUILD_NUMBER}.vsix"
```

Comment:
@jrichter1 Lets rename this vsix file as:
`openshift-${packageJson.version}-${env.BUILD_NUMBER}.vsix `

Reply:
errr, that is the exact same string :)

Reply:
ahh, I updated that.